### PR TITLE
Update Support laravel 7.0

### PR DESCRIPTION
**Pull request by asyrawih**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -65,5 +65,11 @@
     "scripts": {
         "test": "phpunit",
         "cs": "php-cs-fixer fix"
+    },
+    "repositories": [
```
Comment (xar) [inline composer.json:69]: this is not needed

**Comment by feralheart** [comment]
Hi,
Thank you for your contribution. Before I merge it can you do some minor modifications?

File: composer.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     },
     "require-dev": {
         "codeigniter/framework": "~3.0",
-        "illuminate/support": "^5.4",
+        "illuminate/support": "^7.0",
```
Comment (feralheart) [inline composer.json:29]: Don't update exclusively the dependency's major version. It can be a possible project-breaker for some users.

**Comment by feralheart** [comment]
I tested out the current 2.0 branch and the modification for the support is already implemented in #1117

**Pull request closed without merging**